auth_backends/backends.py

```diff
@@ -19,6 +19,9 @@ class EdXOpenIdConnect(OpenIdConnectAuth):
     REDIRECT_STATE = False
     ID_KEY = 'preferred_username'
 
+    # Store the token type to ensure that we use the correct authentication mechanism for future calls.
+    EXTRA_DATA = OpenIdConnectAuth.EXTRA_DATA + ['token_type']
```

Comment:
I spent over an hour trying to find a way to test this, and ended up sad and writing this comment.